packages/host/app/components/ai-assistant/toast.gts

```diff
@@ -176,7 +182,12 @@ export default class AiAssistantToast extends Component<Signature> {
             lastMessage[1].eventId,
           ),
       )[0] ?? null;
-    if (lastMessage) {
+    let fifteenMinutesAgo = subMinutes(new Date(), 15);
+    if (
+      lastMessage &&
+      (isEqual(lastMessage[1].created, fifteenMinutesAgo) ||
```

Comment:
Perhaps, we just need to check if the last message was after fifteenMinutesAgo -- remove the isEqual?

Reply:
agreed